core/chains/evm/config/toml/defaults/Polygon_Zkevm_Cardona.toml

```
@@ -13,12 +13,11 @@ ContractConfirmations = 1
 ResendAfterThreshold = '3m'
 
 [GasEstimator]
+Mode = 'FeeHistory'
+Feehistory.CacheTimeout = '5s'
 PriceMin = '1 mwei'
 BumpPercent = 40
 BumpMin = '20 mwei'
```

Think we should put the `FeeHistory` config in it's own section like BlockHistory
```suggestion
[GasEstimator]
Mode = 'FeeHistory'
PriceMin = '1 mwei'
BumpPercent = 40
BumpMin = '20 mwei'

[GasEstimator.FeeHistory]
CacheTimeout = '5s'
```